src/main/java/com/twilio/oai/TwilioGoGenerator.java

```diff
@@ -64,11 +67,10 @@ public Map<String, Object> postProcessModels(final Map<String, Object> objs) {
 
     @SuppressWarnings("unchecked")
```

Comment:
Don't think this is needed anymore. Recommend installing the `SonarLint` IntelliJ plugin. It will highlight such items (and lots of other cool stuff).